### PR TITLE
etcd Operator upgrade issue for 4.4 RNs

### PR DESCRIPTION
**Pull request by codyhoag**
https://bugzilla.redhat.com/show_bug.cgi?id=1830789

Preview: http://file.rdu.redhat.com/~choag/050220/rn-bz-addition/release_notes/ocp-4-4-release-notes.html#ocp-4-4-known-issues

FYI @vikram-redhat

### CODE REVIEW

**Comment by codyhoag** [comment]
@hexfusion can you verify this known issue description for the upcoming 4.4 release notes?

**Review by hexfusion — COMMENTED** [review]
This fix would assume that `cluster-etcd-operator` was in a degraded state as the result of InstallerPod failure. By forcing a new revision the hope is that transient failure which resulted in the installer pod failure is now overcome. Thus resulting in a successful revision deployment clearing degraded status.

/lgtm

File: release_notes/ocp-4-4-release-notes.adoc [hunk 1]
```diff
@@ -2110,7 +2110,24 @@ issue by deleting the Elasticsearch deployments in sequence.
...
+$ oc patch etcd cluster -p='{"spec": {"forceRedeploymentReason": "recovery-'"$( date --rfc-3339=ns )"'"}}' --type=merge
+----
++
+. Verify the nodes are at the latest revision:
++
+----
+$ oc get etcd '-o=jsonpath={range .items[0].status.conditions[?(@.type=="NodeInstallerProgressing")]}{.reason}{"\n"}{.message}{"\n"}'
+----
++
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1830409[*BZ#1830409*])
```
Comment (wking) [inline release_notes/ocp-4-4-release-notes.adoc:2130]: We now have a 4.4.z clone in https://bugzilla.redhat.com/show_bug.cgi?id=1830789 that these 4.4 notes should probably link instead.

**Comment by codyhoag** [comment]
I've made the recommended updates and am now linking to [BZ#1830789](https://bugzilla.redhat.com/show_bug.cgi?id=1830789). Let me know if anything else needs adjusting here. Thanks!

File: release_notes/ocp-4-4-release-notes.adoc [hunk 2]
```diff
@@ -2110,7 +2110,27 @@ issue by deleting the Elasticsearch deployments in sequence.
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=1824006[*BZ#1824006*])
 
 * {product-title} 4.4 is not shipping with a v.4.4 Metering Operator. Customers can install or continue to run the v.4.3 Metering Operator on {product-title} 4.4 clusters.
-(link:https://bugzilla.redhat.com/show_bug.cgi?id=1829035[*BZ1829035*])
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1829035[*BZ#1829035*])
+
+* When updating an {product-title} cluster from version 4.3 to 4.4, the
+`cluster-etcd-operator` sometimes fails to upgrade because it is in a degraded
```
Comment (wking) [inline release_notes/ocp-4-4-release-notes.adoc:2116]: It's the `etcd` ClusterOperator.  Might be worth making this "...the `etcd` cluster operator..."?  But folks will probably understand either way.

File: release_notes/ocp-4-4-release-notes.adoc [hunk 3]
```diff
@@ -2110,7 +2110,27 @@ issue by deleting the Elasticsearch deployments in sequence.
...
+state. This is caused by an InstallerPod failure. As a workaround, you must
+force a new revision on etcd to overcome the InstallerPod failure, which clears
+the degraded status of the etcd Operator:
```
Comment (wking) [inline release_notes/ocp-4-4-release-notes.adoc:2119]: However you refer to the operator above, probably just echo that here.  Or reword with something like:

> As a workaround, forcing a redeployment allows the operator to recover.

**Comment by codyhoag** [comment]
@wking I've made those adjustments. Please let me know if there's anything else I missed. Thanks!

**Review by mffiedler — APPROVED** [review]
Verified the steps as documented.   Looks great.

/lgtm

**Comment by codyhoag** [comment]
Confirmed with Vikram to move forward with merging. Thanks all!

**Pull request merged**